src/style/CommitBox.ts

```diff
@@ -109,5 +109,6 @@ export const activeStyle = style({
 export const disabledStyle = style({
   cursor: 'not-allowed !important',
   color: 'var(--jp-ui-font-color2) !important',
-  backgroundColor: 'var(--jp-layout-color3) !important'
+  backgroundColor: 'var(--jp-layout-color3) !important',
+  pointerEvents: 'auto !important'
```

Comment:
This is a limitation of the type. It seems it knows about `auto` but not with the flag `!important`. so let's trick ts on this one with a comment:

```suggestion
  // TypeScript does not know about the value with `!important` flag
  pointerEvents: 'auto !important' as any
```